src/stamps/entries.js

```diff
@@ -32,13 +32,14 @@ const stamp = stampit()
 .methods({
   /**
    * Get all the content entries, based on the given parameters.
-   * **DO NOT** use several of id, alias and typeId at the same time - behaviour would be ungaranteed.
+   * **DO NOT** use several of id, alias, typeId and typeAlias at the same time - behaviour would be ungaranteed.
    * @param {object} [params] a parameters object
    * @param {boolean} [params.preview] when true, get the preview version
    * @param {string|date} [params.at] when given, get the version at the given time
    * @param {array} [params.id] an array of entry ids (strings)
    * @param {array} [params.alias] an array of entry aliases (strings)
    * @param {string} [params.typeId] only return entries matching this type id
```

Comment:
As far as I know, all of them except typeAlias handles multiple values now. There's a ticket to make that one also take an array ([APPGRID-5172](https://accedobroadband.jira.com/browse/APPGRID-5172)) but it hasn't been implemented yet.